manifests/crds/argoproj.io_applicationsets.yaml

```diff
@@ -14,7 +12,7 @@ spec:
     listKind: ApplicationSetList
     plural: applicationsets
     singular: applicationset
-  scope: Namespaced
+  scope: ""
```

Comment:
ApplicationSet should be Namespaced shouldn't it? (similar to Application)

Reply:
Yes，it should Namespaced,  I run `make manifest` cmd,  it will remvoe `Namespced` line auto,  I don't know why.

I will add it back.

Reply:
:+1: 

That is weird, I thought it was "// +genclient:nonNamespaced" to make a global CR but we don't have that anywhere. We'll have to keep an eye on that.